Found the problem. It's missing gfortran
```
   stderr: hostname: Name or service not known
gmake: gfortran: No such file or directory
gmake: gfortran: No such file or directory
hostname: Name or service not known
gmake: gfortran: No such file or directory
gmake: gfortran: No such file or directory
gmake: gfortran: No such file or directory
gmake: *** [/data/mash/cche/azp-agent-in-docker-cuda/azp-agent-avatargpu/regression-tests/amrex//Tools/GNUMake/Make.rules:329: tmp_build_dir/o/3d.gnu.EXE/AMReX_fort_mod.o] Error 127
gmake: *** Waiting for unfinished jobs....
```

--- a/.devcontainer/cuda-azp-agent-in-docker/Dockerfile
+++ b/.devcontainer/cuda-azp-agent-in-docker/Dockerfile
@@ -5,7 +5,7 @@ ENV TARGETARCH="linux/amd64"
 USER root
 RUN apt update && \
   apt upgrade -y && \
-  apt install -y curl git jq libicu74 python3-h5py
+  apt install -y curl git jq libicu74 python3-h5py gfortran
 
 # Install h5py
 # RUN pip3 install h5py 

--- a/.devcontainer/cuda-azp-agent-in-docker/build.sh
+++ b/.devcontainer/cuda-azp-agent-in-docker/build.sh
@@ -1,3 +1,6 @@
+#!/bin/bash
+set -e
+
 export DOCKER_BUILDKIT=1
 docker buildx create --use
 

